src/mococrw/openssl_wrap.h

```diff
@@ -276,6 +303,14 @@ enum class X509Extension_NID : int {
     SubjectKeyIdentifier = NID_subject_key_identifier
 };
 
+enum class RSAPaddingMode
+{
+    NONE = RSA_NO_PADDING,
```

Comment:
Why support RSA with no padding at all? It is considered to be not secure and therefore its usage is not recommended.

Reply:
Agreed, but It might be useful in the future in order to combine RSA with block ciphers or something else. And in the ens it's up to the user to decide.

Reply:
In my view it doesn't make sense to allow a deprecated and insecure algorithm to be used.